lib/pa_ess/http_updater.ex

```diff
@@ -165,8 +165,10 @@ defmodule PaEss.HttpUpdater do
   def process({:send_audio, [{station, zones}, audios, priority, timeout]}, state) do
     case audios do
       list when is_list(list) ->
-        for audio <- list do
-          process_send_audio(station, zones, audio, priority, timeout, state)
+        combined_audios = combine_audio(list)
```

Comment:
Ah, didn't realize that the list version was used by the bus code. Agreed that it should happen further up the stack. It should certainly be doable, but there's some friction since it's a departure from Realtime Signs' typical pattern where the structs get converted to message/take ids at basically the last possible second.

Certainly something to consider pursuing though if/when we come back to this task